test/schemas.js

```diff
@@ -36,6 +37,46 @@ describe('schemas', () => {
     assert.deepEqual(schema.title, 'Data Resource')
   })
 
+  it('data-resource path property', async () => {
+
+    // Valid paths
+    const schema = require('../build/schemas/data-resource.json')
+    const invalidPaths = [
+      '../dest',
+      './dest',
+      '/dest',
+      '~/dest',
+      'dest/../../bad',
```

Comment:
@anuveyatsu 
If I'm doing everything right this test fails for me.

It seems regexp should be extended to forbid `..` at any position.